src/main/array/ArrayAndLoop.java

```diff
@@ -0,0 +1,102 @@
+package main.array;
+
+public class ArrayAndLoop {
+        public static void main(String[] args) {
+            iphoneLength();
+            getConcat();
+            getTrim();
+            getEndWith();
+            getToUpperCase();
+            getToLowerCase();
+            getEquals();
+
+        }
+
+    private static void iphoneLength() {
+            System.out.println("\nLENGTH:");
+            String[] iphones = {"Iphone 11", "Iphone 12", "Iphone 13"};
+            int phone = 0;
+
+            for (int i = 0; i < iphones.length; i++) {
+                phone = phone + iphones[i].length();
+
+            }
+            System.out.println("The Length would be: " + phone);
+
+        }
+
+        private static void getConcat() {
+            System.out.println("\nCONCAT:");
+            String[] conTask = new String[3];
+
+            conTask[0] = " I didn't know that ";
+            conTask[1] = "learning java";
+            conTask[2] = " can be this much fun.";
+            String result = (conTask[0].concat(conTask[1].concat(conTask[2])));
+
+            for (int i = 0; i < 1; i++) {
+                System.out.println(i + result);
+            }
+
+        }
+
+        private static void getTrim() {
+            System.out.println("\nTRIM:");
+            String[] trimTask = {"  Create  ", "New     ", "  Project    "};
+
+            for (int i = 0; i < 3; i++) {
+                System.out.println("Trimed: " + trimTask[i].trim());
+            }
+
+        }
+
+        private static void getEndWith() {
+            System.out.println("\nENDWITH:");
+            String[] endWTask = {"Having ", "Fun With ", "Java"};
+
+            if (endWTask[2].endsWith("a")) {
+                System.out.println("Good: ");
+            } else {
+                System.out.println("Wrong");
+            }
+
+        }
+
+        private static void getToUpperCase() {
+            System.out.println("\nTO UPPER CASE:");
+            String[] seasons = {" Winter", " Spring ", "Summer", " Autumn"};
+
+            for (int i = 0; i < 1; i++) {
```

Comment:
please read the task one more time
you should rewrite every second element from the array.
After that you should output the result outside of the fori loop/